envoy/api/v2/cds.proto

```diff
@@ -356,6 +356,16 @@ message Cluster {
     // .. note::
     //   The specified percent will be truncated to the nearest 1%.
     core.Percent healthy_panic_threshold = 1;
+    // [#not-implemented-hide:] Configures percentage of requests that will be routed to same zone
+    // If not specified, the default is 100%.
+    // :ref:`runtime values <config_cluster_manager_cluster_runtime_zone_routing>`.
```

Comment:
Can you take a look at how this renders in the docs when ` [#not-implemented-hide:]` is not there? I think it will look a little strange, as it will just be two additional sentences with links. I think you want to use RST markdown, e.g.
```
If not specified the default is 100%.
* :ref:`runtime values <config_cluster_manager_cluster_runtime_zone_routing>`
* :ref:`Zone aware routing support <arch_overview_load_balancing_zone_aware_routing>`
```

Reply:
Thanks for the review @htuch  . I can move description to next line and push .